job-server/src/main/scala/spark/jobserver/JobManager.scala

```diff
@@ -12,54 +12,46 @@ import org.slf4j.LoggerFactory
 import spark.jobserver.common.akka.actor.ProductionReaper
 import spark.jobserver.io.{JobDAO, JobDAOActor}
 import scala.collection.JavaConverters._
+import scala.util.Try
 
 /**
  * The JobManager is the main entry point for the forked JVM process running an individual
- * SparkContext.  It is passed $workDir $clusterAddr $configFile
- *
- * Each forked process has a working directory with log files for that context only, plus
- * a file "context.conf" which contains context-specific settings.
+ * SparkContext.  It is passed $clusterAddr $actorName $systemConfigFile
```

Comment:
Thanks for your great work. I am not sure if this $systemConfigFile is still necessary now after transfer configuration using akka message. Because In mesos cluster mode, the $CONF_FILE in manager_start.sh is set to be a remote address, but In JobManager we still read it like a normal local file using `val systemConfigFile = new File(args(2))`